pkg/controllers/termination/terminate.go

```diff
@@ -101,18 +109,13 @@ func (t *Terminator) terminate(ctx context.Context, node *v1.Node) error {
 
 // evictPods returns true if there are no evictable pods
 func (t *Terminator) evictPods(ctx context.Context, pods []*v1.Pod) bool {
-	empty := true
```

Comment:
Does this function even need to return true anymore? Shouldn't it return err? We already know if the node is empty before we even call this function, since we can check len(pods) before we call this.

Reply:
I think it's a matter of doing a len(check) before, or doing it as the return value. Either way, we still need to check if there was a pod that needs evicting. I added the return `len(pods)==0` so that if we call evictPods, it'll just return that immediately because the for loop would have nothing to iterate through.

With regards to erroring vs bool, I think we decided on an early comment to punt on this to another PR.